test/cpp/end2end/channelz_service_test.cc

```diff
@@ -118,8 +122,14 @@ class Proxy : public grpc::testing::EchoTestService::Service {
     return stream_to_backend->Finish();
   }
 
+  std::shared_ptr<Channel> channel(int i) {
```

Comment:
Doesn't look like this is actually being used anywhere.  Is this left over from an abandoned approach?

Reply:
Ah!  Sorry, I missed that.